examples/basic_example/graph_client_basic_example.go

```diff
@@ -21,6 +22,8 @@ const (
 	port     = 3699
 	username = "root"
 	password = "nebula"
+	useSSL   = false
```

Comment:
Please make sure these changes in examples will not be merged into the master branch, the examples should be concise.

Reply:
Adding this doesn't make it too complicated, but he can tell users how to use ssl and http2. I think this is also beneficial? What do you think?

Reply:
Of course, this will not be merged into master for the time being. If you think it is not necessary, I will delete it before merging into master.